packages/components/ResultDisplay.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Grid, Text } from '@chakra-ui/react';
+
+const positions = [
+  {
+    candidate: 'Daniel Menezes',
+    vice: 'Joel Morais',
+    partido: 'PCdoB',
+    porcentagem: '55%',
+    votos: '550',
+  },
+  {
+    candidate: 'Gabriel Amarante',
+    vice: 'Érica Poline',
+    partido: 'PT',
+    porcentagem: '10%',
+    votos: '100',
+  },
+  {
+    candidate: 'Lucas Moreira',
+    vice: 'Raquel',
+    partido: 'Psol',
+    porcentagem: '35%',
+    votos: '350',
+  },
+];
+export default function ResultDisplay() {
+  return (
+    <Grid h="50vh" w="100wh" alignSelf="center" justifyContent="center">
```

Comment:
Still a grid here, As I've metioned, you're not using grid feature so why a grid? Also, this page should render multiple cards as the result, not a fixed number, so I beliave a flex structure would be a better fit

Reply:
agree

Reply:
When the flexible structure, did not understand. You had talked about this mock list.